mappers/contact-info.js

```diff
@@ -92,7 +92,7 @@ function mapProductOwner(recordId, record, postfix = '') {
     if (title) {
       contactStatements.push(new Statement(sym(contactPointUri), VCARD('honorific-prefix'), lit(title, 'nl')));
     } else {
-      console.error(`Cannot map product owner title value '${record[`product_owner_title${postfix}`]}' for record ${recordId}`);
+      errorLogger('product owner title', record[`product_owner_title${postfix}`], recordId);
```

Comment:
Pass the same field name (`product_owner_title${postfix}`) as used to lookup the value as first argument. I know it was different before, but old code is wrong.